.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+COVERALLS_REPO_TOKEN: pujZRDBed3JlLSQ79HwLPSbDl0vBSFkR4
```

Comment:
Hi @Kasulejoseph, as pointed out by James, please remove coveralls config from the repo. since we're set on using code climate for coverage and maintainability badges.

Reply:
thanks, I have removed it